js/weather-service.js

```diff
@@ -25,7 +28,7 @@
                 return null;
             }
             service.forcast.data.currently.day = moment.unix(service.forcast.data.currently.time).format('ddd')
-            service.forcast.data.currently.temperature = parseFloat(service.forcast.data.currently.temperature).toFixed(1);
+            service.forcast.data.currently.temperature = vsprintf(numberFormat, [parseFloat(service.forcast.data.currently.temperature).toFixed(1)]);
```

Comment:
This seems like a pointless dependency to me...
Let's say `service.forcast.data.currently.temperature` = 123.45
``` javascript
parseFloat(service.forcast.data.currently.temperature).toFixed(1); //123.5
vsprintf(numberFormat, [parseFloat(service.forcast.data.currently.temperature).toFixed(1)]) //123.5
```

Why do we need this?